token/services/drivers/dbholder.go

```diff
@@ -7,12 +7,14 @@ SPDX-License-Identifier: Apache-2.0
 package drivers
```

Comment:
can we move this package under db?

Reply:
I can move the db drivers. but the holders contain generic drivers too